css/css-fonts/parsing/font-variant-alternates-invalid.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Fonts Level 3: parsing font-variant-alternates with invalid values</title>
```

Comment:
Please change the font-variant-alternates tests to Fonts 4. The CSS WG resolved to move this feature from Fonts 3 to Fonts 4, as there is currently only one implementation. (the spec edits are pending on me, hope to get them done before the CSS f2f in Sydney).